docs/reference/toc.yml

```
@@ -0,0 +1,13 @@
+toc:
+  - file: index.md
+    children:
+      - file: setup/index.md
+        children:
+          - file: setup/k8s.md
+          - file: setup/manual-instrumentation.md
+      - file: configuration.md
+      - file: supported-technologies.md
+      - file: migration.md
+      - file: overhead.md
+      - title: Troubleshooting
+        crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/python/index.md
```

I think you need to flatten the nav structure (similar to https://github.com/elastic/elastic-agent/pull/9953#pullrequestreview-3226598623):

```suggestion
toc:
  - file: index.md
  - file: setup/index.md
    children:
      - file: setup/k8s.md
      - file: setup/manual-instrumentation.md
  - file: configuration.md
  - file: supported-technologies.md
  - file: migration.md
  - file: overhead.md
  - title: Troubleshooting
    crosslink: docs-content://troubleshoot/ingest/opentelemetry/edot-sdks/python/index.md
```